package.json

```diff
@@ -157,11 +164,12 @@
     "merge-stream": "^1.0.0",
     "mock-require": "^3.0.1",
     "nib": "^1.1.0",
-    "react-css-modules": "^3.7.6",
+    "react-css-modules": "^4.7.9",
     "react-input-autosize": "^1.1.0",
-    "react-router": "^2.4.0",
-    "react-router-redux": "^4.0.4",
-    "react-test-renderer": "^15.6.2",
+    "react-test-renderer": "^16.8.6",
+    "redux-devtools": "^3.5.0",
```

Comment:
There is an issue with `redux-devtools`. I'm not sure if we need to fix this before merging but I'm getting the error message from the screenshot pretty often especially if I'm selecting a dom element.

During install of `redux-devtools` I'm getting the following `warning " > redux-devtools@3.5.0" has incorrect peer dependency "react-redux@^4.0.0 || ^5.0.0 || ^6.0.0".`
There is an issue to it [here](https://github.com/reduxjs/redux-devtools/issues/473) but with-out a comment. Maybe we have to change version to `React-redux` 6.x until this is fixed.

![grafik](https://user-images.githubusercontent.com/3046542/57568012-a4ba0100-73e1-11e9-839c-a748030e5d59.png)

Reply:
I can not reproduce the behaviour with the warning message at the moment. Is this still a problem? If not I think we can stay ar react-redux 7.x and ignore the warning from `redux-devtools`.